### PR TITLE
Fix: Dao 692 arbitrum login

### PR DESCRIPTION
**Pull request by WissenIstNacht**
This PR allows the app to connect to Arbitrum.

### CODE REVIEW

File: packages/web-app/src/hooks/useWalletTokens.tsx [hunk 1]
```diff
@@ -1,27 +1,105 @@
...
-      if (account === null) {
+      setIsLoading(true);
+      if (account === null || provider === null) {
         setWalletTokens([]);
         return;
       }
 
-      const tokenList = await getTokenList();
       if (Number(balance) !== -1 && Number(balance) !== 0)
         await tokenList.push(constants.AddressZero);
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useWalletTokens.tsx]: We can remove the await there, yes?

File: packages/web-app/src/hooks/useEnsData.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      }
+      setIsLoading(false);
+    }
+
+    fetchEnsData();
+  }, [web3, address]);
+  return {data: name, isLoading, error};
+}
+
+export function useEnsAvatar(address: Address): HookData<string> {
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useEnsData.tsx:36]: What is the reasoning behind not merging these two into one considering most of the time we'll want both?

**Review by Fabricevladimir — APPROVED** [review]
Thanks for this. Cleaned up things really well

File: packages/web-app/src/hooks/useEnsData.tsx (hunk 2 above)
Comment (WissenIstNacht) [reply packages/web-app/src/hooks/useEnsData.tsx:36]: Right, good question. So there's two things that kinda go hand-in-hand here. The first is, as you said, we wont always use both. But more importantly, having an ens record does not imply having an Avatar. In which case you'll only want the name. Now the second, related reason is data and error handling. I think having them separated makes it simpler to use (although there's a slight overhead of characters) when taking into account things like errors. E.g., what do you return when one call succeeds ant the other fails? Which error do you return when both fail?

Comment (Fabricevladimir) [reply packages/web-app/src/hooks/useEnsData.tsx:36]: Makes sense 👍🏾

**Review by RakeshUP — APPROVED** [review]

**Pull request merged**